<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="stop_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_resume_btn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QTableView" name="table_view">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>1101</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>165</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择协议:</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Pure_aloha</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Slot_aloha</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>黑体</family>
             <pointsize>13</pointsize>
            </font>
           </property>
           <property name="text">
            <string>帧时：</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_2">
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>50ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>100ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>200ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>400ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>900ms</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000ms</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>lambda：</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
